tests/templates/ingress.yaml

```diff
@@ -18,7 +18,7 @@ spec:
         app: microbot
     spec:
       containers:
-      - image: cdkbot/microbot-$ARCH
+      - image: cdkbot/microbot-amd64
```

Comment:
You accidentally committed the `amd64`

Reply:
yes its from the test template,did a correction for it